client/src/components/Form/Elements/FormData/FormData.vue

```diff
@@ -267,7 +271,11 @@ function clearHighlighting(timeout = 1000) {
  */
 function createValue(val?: Array<DataOption> | DataOption | null) {
     if (val) {
-        const values = Array.isArray(val) ? val : [val];
+        let values = Array.isArray(val) ? val : [val];
+
+        // Remove duplicates based on item.id
```

Comment:
I'd definitely like to see minor fixes like that be pulled out for separate evaluation. This seems pretty questionable and maybe hiding a different bug ?

Reply:
Yes this was interesting, it would happen when there would be an emit (from `DefaultBox` or the collection creator) in order to set the `currentValue` _sometimes_. I would've thought duplicates wouldn't be possible in the whole stack of `handleIncoming` all the way to setting `formattedOptions`, but it still happens